client/db.go

```diff
@@ -438,6 +438,19 @@ func (tx *Tx) Get(key interface{}) (KeyValue, error) {
 	return runOneRow(tx, tx.B.Get(key))
 }
 
+// GetProto retrieves the value for a key and decodes the result as a proto
+// message.
+//
+// key can be either a byte slice, a string, a fmt.Stringer or an
+// encoding.BinaryMarshaler.
+func (tx *Tx) GetProto(key interface{}, msg gogoproto.Message) error {
```

Comment:
Would be good to compare the methods on `DB` and `Tx` using reflection. Somewhat harder to add this to `Batch` and `Batcher` as those gets are not synchronous. Suppose I can use the `Call.Post` mechanism. I'll do that in a separate change.